css/main.css

```diff
@@ -60,78 +65,127 @@ p {
     width: 50%;
 }
 
-.description h1 {
+.hero-title {
     color: #af0404;
     font-size: 3rem;
 }
 
-.description p {
-    color: #dedede;
-    font-size: 1.2rem;
-    line-height: 1.5;
+.hero-description {
+    font-size: 1.8rem;
 }
 
-
 /* Community Section*/
 
 .community {
-    background-color: #af0404;
-    padding-bottom: 50px;
+    background-color: var(--main-bg-color);
+    padding-bottom: 5rem;
 }
 
 .text {
-    margin-top: 20%;
+    margin-top: 5rem;
 }
 
 .buttons {
-    margin-top: 20%;
```

Comment:
If you use %, it's okay to keep using them, because they are relative units. There's no need to change them to `rem`.